src/widgets/RainWidget.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import GenericWidget from './GenericWidget';
+import './RainWidget.css';
+
+class RainWidget extends Component {
+
+  render() {
+    return (
+      <GenericWidget class="RainWidget" icon="cloud"
+          isOn={this.props.rain > 0}
+          value1={<div className="rain">{this.props.rain}</div>}
+          value2={<div className="rate">{this.props.rate}</div>}
```

Comment:
You don't need extra divs here, see for example WattWidget. In the css you can simply do:

.RainWidget h2:after {
  content: ' mm';
}

.RainWidget .value2:after {
  content: ' mm/h';
}